examples/js/exchange-capabilities.js

```diff
@@ -8,6 +8,7 @@ const csv = process.argv.includes ('--csv')
     , log = require ('ololog').noLocate
     , ansi = require ('ansicolor').nice
     , sortCertified = process.argv.includes ('--sort-certified') || process.argv.includes ('--certified')
+    , selectedExchanges = process.argv.includes ('--exchanges') ? process.argv.at(process.argv.findIndex ((val)=> val === '--exchanges') + 1) : []
```

Comment:
I'm using `zsh` on MacOS (this is similar to `bash` in Ubuntu), and this command-line syntax might not work for all systems:

```
~/ccxt $ node examples/js/exchange-capabilities.js --exchanges [aax,ftx]
zsh: no matches found: [aax,ftx]
```

I would suggest a slightly different syntax:

```JavaScript
~/ccxt $ node examples/js/exchange-capabilities.js --exchanges=aax,ftx
zsh: no matches found: [aax,ftx]
```

```suggestion
    , exchangesArgument = process.argv.find (arg => arg.startsWith ('--exchanges='))
    , exchangesArgumentParts = exchangesArgument ? exchangesArgument.split ('=') : []
    , selectedExchanges = (exchangesArgumentParts.length > 1) ? exchangesArgumentParts[1].split (',') : []
```